scss/main/main.scss

```diff
@@ -850,4 +866,186 @@ footer {
       float: left;
     }
   }
-}
\ No newline at end of file
+}
+
+@media (max-width: 1159px) {
+
+  p {
+    font-size: 14px;
+  }
+
+  h3 {
+    font-size: 34px;
+  }
+
+  .wrapper {
+    min-width: 680px;
+  }
+
+  .container {
+    width: 100%;
+    margin: 0;
+  }
+
+  nav {
+
+    .container {
+
+      ul {
+
+        li {
+
```

Comment:
Extra blank lines between selectors. If there are no rules between `.container` and `ul` blank line isn't needed